File: app/src/main/java/com/example/mad2/pagetwo/PageTwoVM.java
package com.example.mad2.pagetwo;

import android.os.Bundle;
import androidx.lifecycle.MutableLiveData;
import androidx.lifecycle.ViewModel;

public class PageTwoVM extends ViewModel {
    public MutableLiveData<PageTwoModel> pageTwoModel = new MutableLiveData<>(new PageTwoModel());
    private MutableLiveData<Bundle> navArguments = new MutableLiveData<>();

    public MutableLiveData<PageTwoModel> getPageTwoModel() {
        return pageTwoModel;
    }

    public MutableLiveData<Bundle> getNavArguments() {
        return navArguments;
    }

    public void setNavArguments(Bundle navArguments) {
        this.navArguments.setValue(navArguments);
    }
}

File: jia_hui_s_application/app/src/main/java/com/jiahuisapplication/app/modules/pagetwo/data/viewmodel/PageTwoVM.kt
package com.jiahuisapplication.app.modules.pagetwo.`data`.viewmodel

import android.os.Bundle
import androidx.lifecycle.MutableLiveData
import androidx.lifecycle.ViewModel
import com.jiahuisapplication.app.modules.pagetwo.`data`.model.PageTwoModel
import org.koin.core.KoinComponent

class PageTwoVM : ViewModel(), KoinComponent {
  val pageTwoModel: MutableLiveData<PageTwoModel> = MutableLiveData(PageTwoModel())

  var navArguments: Bundle? = null
}
